application.tf

```diff
@@ -318,6 +318,29 @@ YAML
   }
 }
 
+module "karpenter" {
```

Comment:
Is there a reason you chose to put this in the application.yaml rather than in nodes.tf?

Reply:
Yes, Karpenter is an application with deployment, crds and objects. So following the pattern we adopted I believe makes sense to be inside application file